roles/confluent.common/tasks/confluent_cli.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Confluent CLI system check
+  assert:
+    that: ansible_facts['system']|lower == "linux"
+    fail_msg: "Confluent CLI system not supported"
+    quiet: true
+
+- name: Confluent CLI architecture check
+  assert:
+    that: confluent_cli_goarch[ansible_facts['architecture']] is defined
+    fail_msg: "Confluent CLI architecture not supported"
+    quiet: true
+
+- name: Confluent CLI create base path
+  file:
+    path: "{{confluent_cli_base_path}}"
+    state: directory
+    mode: '0755'
+
+- name: Confluent CLI create executable directory
```

Comment:
Is this task necessary, I'm wondering if the unarchive will create it

Reply:
No, if I try to unarchive in a non existing folder I get:
`fatal: [localhost]: FAILED! => {"changed": false, "msg": "dest '/tmp/notexistdir' must be an existing dir"}`